product/views.py

```diff
@@ -134,3 +135,280 @@ def get_user_info(community):
+
+    @login_decorator(view_name='CommunityView')
+    def get(self, request, id):
+        try:
+            community      = Community.objects.select_related('user','product','product__sub_category','product__creator').prefetch_related('user__community_like','user__community_like').get(pk=id)
+            comments       = CommunityComment.objects.select_related('user').filter(community_id=id)
+            comment_count  = CommunityComment.objects.filter(community_id=id).count()
+            community_like = CommunityLike.objects.filter(community_id=id)
+
+            community_detail = {
+                'id'           : community.id,
+                'name'         : community.user.name,
+                'profileImage' : community.user.profile_image,
+                'createdAt'    : community.created_at,
+                'content'      : community.description,
+                'likeCount'    : community.user.community_like.all().count(),
+                'isLiked'      : get_is_like(request.user.id) if request.user else False,
+                'commentCount' : comment_count,
+                'thumbnail'    : community.product.thumbnail_image,
+                'subCategory'  : community.product.sub_category.name,
+                'creator'      : community.product.creator.name,
+                'productTitle' : community.product.name,
+                }
+
+            comment_list = [{
+                'id'           : comment.id,
+                'name'         : comment.user.name,
+                'profileImage' : comment.user.profile_image,
+                'created_at'   : comment.created_at,
+                'content'      : comment.content 
+                } for comment in comments ]
+
+            return JsonResponse({"community_detail": community_detail, "comment_count": comment_count, "comment_list": comment_list}, status=200)
+        except Community.DoesNotExist:
+            return JsonResponse({'MESSAGE':'NO_EXIST_COMMUNITY'}, status=400)
+        except KeyError as e :
+            return JsonResponse({'MESSAGE': f'KEY_ERROR:{e}'}, status=400)
+        except json.JSONDecodeError as e :
+            return JsonResponse({'MESSAGE': f'JSON_DECODE_ERROR:{e}'}, status=400)
+
+def get_is_like(user_id):
+    return CommunityLike.objects.filter(community_id=id, user_id =user_id).exists()
+
+
+class CommunityCommentView(View):
+    @login_decorator(view_name='CommunityCommentView')
+    def post(self, request):
+        data=json.loads(request.body)
+        try:
+            user      = request.user
+            community = Community.objects.get(pk=data['community_id'])
+           # comment   = CommunityComment.objects.get(pk=data['community_comment_id'])
+            content   = data['content']
+
+            CommunityComment.objects.create(
+                user      = user,
+                communtiy = community,
+                content   = content,
+            )
```

Comment:
```suggestion
            user      = request.user
            content   = data['content']

            CommunityComment.objects.create(
                user      = user,
                communtiy_id = data['community_id'],
                content   = content,
            )
```
혹시 community_id가 FK로만 동작하는 역할이라면, 클래스를 get해오지 않고 바로 사용하시는 방법을 추천드립니다.
`community`라는 변수 선언도 하지 않아도 되겠네요 :)

Reply:
네! FK로만 동작하는 역할이였습니다! 감사합니다.
저렇게 바로 사용하는것보다 저장하고 쓰는게 더 효율적인줄 알았습니다